10up/ElasticPress
# Fix: Thumbnails are not removed from indexed WooCommerce Products whe…

### Description of the Change

This PR fixes the issue where removing the thumbnails doesn't update the indexed WooCommerce Product. The issue was that query only look for the post where it should look for all the indexable posts.


Closes #3254

### How to test the Change


1. Upload an image attachment.
2. Create two Products, set the uploaded attachment as the featured image for both.
3. Check the indexed Products, they should include metadata containing the thumbnail ID.
4. Delete the attachment.
5. Check the indexed Products, they should not have the thumnail meta


### Changelog Entry

> Fixed - Thumbnails are not removed from indexed WooCommerce Products when the attachments are deleted


### Credits

Props @burhandodhy @JakePT 


### Checklist:
